tests/test_mqtt_block.py

```diff
@@ -1,19 +1,75 @@
+from unittest.mock import patch, MagicMock
 from nio.block.terminals import DEFAULT_TERMINAL
 from nio.signal.base import Signal
 from nio.testing.block_test_case import NIOBlockTestCase
-from ..example_block import Example
+from ..mqtt_publish_block import MqttPublish
+from ..mqtt_subscribe_block import MqttSubscribe
 
+import paho.mqtt.client as mqtt
```

Comment:
Similar comment for some of the other files: we don't need a blank line between imports. I'm not sure why we do that in some places. I don't think it's recommended.